lib/content-services/src/lib/search/services/search-query-builder.service.spec.ts

```diff
@@ -15,6 +15,8 @@
  * limitations under the License.
  */
 
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
This will hide eslint issues for any in whole file so we won't see any problems here. Instead adding that comment, could you do following:
Create separate task in jira to replace any with correct types in old code of that file

Then it will be better to fix those places rather than hide problems. 

If you prefer you can fix old places in that PR but I'm fine to create separate task for that as well.